app/server.go

```diff
@@ -0,0 +1,21 @@
+package app
+
+import (
+	"github.com/gorilla/mux"
```

Comment:
By convention, the imports should be separated as follows:

import (
    Go core
    
    Local
    
    ThirdParty
)
Do this in all the imports.